CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ find_package(OMECompat 5.2.0 REQUIRED)
 find_package(OMECommon 5.2.0 REQUIRED)
 find_package(OMEXML 5.2.0 REQUIRED)
 find_package(OMEFiles 0.2.3 REQUIRED)
-find_package(BioFormatsJACE 5.2.4)
```

Comment:
I think the issue was related to the JACE discovery and the `CMAKE_PATH_PREFIX` variable. After removing the `sed` workaround, I was able to rebuild everything and execute the JACE scripts.